src/index.ts

```diff
@@ -27,11 +27,11 @@ async function createOwaspSuppressions() {
   for (const line of lines) {
     const res = reg.exec(line);
 
+    if (!res?.[0] || line === "") continue; // ignore the line
```

Comment:
Add a trim to avoid lines with empty spaces too

```suggestion
    if (!res?.[0] || line.trim() === "") continue; // ignore the line
```